x-pack/plugins/security_solution_serverless/public/get_started/card_step/step_content.tsx

```diff
@@ -8,94 +8,122 @@
+} from '../types';
 
 const LEFT_CONTENT_PANEL_WIDTH = 486;
 const RIGHT_CONTENT_PANEL_WIDTH = 510;
 const RIGHT_CONTENT_HEIGHT = 270;
 const RIGHT_CONTENT_WIDTH = 480;
 
 const StepContentComponent = ({
+  autoCheckIfStepCompleted,
+  cardId,
   description,
   hasStepContent,
   isExpandedStep,
+  indicesExist,
+  sectionId,
   splitPanel,
   stepId,
+  toggleTaskCompleteStatus,
 }: {
+  autoCheckIfStepCompleted?: CheckIfStepCompleted;
+  cardId: CardId;
   description?: React.ReactNode[];
   hasStepContent: boolean;
   isExpandedStep: boolean;
+  indicesExist: boolean;
+  sectionId: SectionId;
   splitPanel?: React.ReactNode;
-  stepId: string;
+  stepId: StepId;
+  toggleTaskCompleteStatus: ToggleTaskCompleteStatus;
 }) => {
   const { euiTheme } = useEuiTheme();
   const shadow = useEuiShadow('s');
 
+  useCheckStepCompleted({
+    autoCheckIfStepCompleted,
+    cardId,
+    indicesExist,
+    sectionId,
+    stepId,
+    toggleTaskCompleteStatus,
+  });
+
   return hasStepContent && isExpandedStep ? (
-    <>
-      <EuiFlexGroup
-        color="plain"
-        css={css`
-          margin-top: 20px;
-          margin-left: ${euiTheme.size.l};
-          transition: opacity ${euiTheme.animation.normal};
-          overflow: hidden;
-        `}
-        data-test-subj={`${stepId}-content`}
-        direction="row"
-        gutterSize="none"
-      >
-        {description && (
-          <EuiFlexItem
-            grow={false}
-            css={css`
-              padding: 0 ${euiTheme.size.l} 0 ${euiTheme.size.s};
-              width: ${LEFT_CONTENT_PANEL_WIDTH}px;
-            `}
-          >
-            <EuiText size="s">
-              {description?.map((desc, index) => (
-                <p
-                  data-test-subj={`${stepId}-description-${index}`}
-                  key={`${stepId}-description-${index}`}
-                  className="eui-displayBlock"
-                  css={css`
-                    margin-bottom: ${euiTheme.base * 2}px;
-                    margin-block-end: ${euiTheme.base * 2}px !important;
-                    line-height: ${euiTheme.size.l};
-                  `}
-                >
-                  {desc}
-                </p>
-              ))}
-            </EuiText>
-          </EuiFlexItem>
-        )}
-        {splitPanel && (
-          <EuiFlexItem
-            grow={false}
-            data-test-subj="split-panel"
-            css={css`
-              padding: 0 6px 0 ${euiTheme.size.l};
-              width: ${RIGHT_CONTENT_PANEL_WIDTH}px;
-            `}
-          >
-            {splitPanel && (
-              <div
+    <EuiFlexGroup
+      color="plain"
+      css={css`
+        justify-content: space-between;
+        margin-top: ${euiTheme.size.l};
+        padding: ${euiTheme.size.l};
+        transition: opacity ${euiTheme.animation.normal};
+        overflow: hidden;
+        border: 1px solid ${euiTheme.colors.lightShade};
+        border-radius: ${euiTheme.border.radius.medium};
```

Comment:
Same. 
Also, considering the ammount of custom CSS we have in the get_started dir, it would also be nice extracting it to separate files, like `step_content.styles.ts`.